peps/pep-0013.rst

```diff
@@ -105,9 +105,11 @@ A council election consists of two phases:
   must be nominated by a core team member. Self-nominations are
   allowed.
 
-* Phase 2: Each core team member can vote for zero or more of the
-  candidates. Voting is performed anonymously. Candidates are ranked
-  by the total number of votes they receive. If a tie occurs, it may
+* Phase 2: Each core team member can assign zero to five stars to each
+  candidate. Voting is performed anonymously. The outcome of the vote
+  is determined using the STAR voting system (www.starvoting.org),
+  modified to use the "Multi-winner Bloc STAR" approach
+  (www.starvoting.org/multi_winner). If a tie occurs, it may
```

Comment:
What about using actual hyperlinks:

```suggestion
  is determined using the `STAR voting system <https://www.starvoting.org/>`__,
  modified to use the `Multi-winner Bloc STAR <https://www.starvoting.org/multi_winner>`__)
  approach. If a tie occurs, it may
```

Reply:
Yeah, that makes sense. Thanks for the idea!